packages/plugin-npm-cli/sources/commands/npm/audit.ts

```diff
@@ -131,12 +131,12 @@ export default class NpmAuditCommand extends BaseCommand {
         registry,
       }) as unknown as Promise<npmAuditTypes.AuditResponse>;
 
-      const deprecations = await Promise.all(this.noDeprecations ? [] : Array.from(packages, async ([packageName, versions]) => {
+      const deprecations = await Promise.all(this.noDeprecations ? [] : Array.from(Object.entries(payload), async ([packageName, versions]) => {
```

Comment:
While at it, why not lift `this.noDeprecations` check and avoid Promise.all call?
 
```suggestion
      const deprecations = this.noDeprecations ? [] : await Promise.all(Array.from(Object.entries(payload), async ([packageName, versions]) => {
```